testslide/runner.py

```diff
@@ -180,6 +180,42 @@ def print_cyan(self, *values, **kwargs):
 
 
 class FailurePrinterMixin(ColorFormatterMixin):
+    def _print_stack_trace(self, exception: BaseException, cause: int) -> None:
+        indent = "  " * cause
+        if cause:
+            self.print_red(f"\n    {indent}Caused by ", end="")
+
+        self.print_red(
+            "{exception_class}: {message}".format(
+                exception_class=exception.__class__.__name__,
+                message="\n    ".join(str(exception).split("\n")),
```

Comment:
I always get confused by indents xd, shouldn't there be an `{indent}` here too?

Reply:
Yeah, this was wrong, good catch!